libraries/AP_Motors/AP_Motors6DOF.cpp

```diff
@@ -338,6 +344,43 @@ void AP_Motors6DOF::output_armed_stabilizing()
             }
         }
     }
+
+	// Current limiting
+    if (_batt_current_max <= 0.0f) {
+        return;
+    }
+
+    AP_BattMonitor &battery = AP::battery();
```

Comment:
shouldn't some of these be defined `const` just to be on the safe side ?

Reply:
Done. None of the others should be const.